src/base/IoManip.h

```diff
@@ -13,7 +13,6 @@
 
```

Comment:
The PR description is very long. I am not sure the future commit message is the right place to document how ADL works (in the context of this problem). Should I remove most of that description and simply refer to ADL as a well-understood concept (despite our obviously insufficient ADL understanding, at least prior to this work)?